adafruit_platformdetect/board.py

```diff
@@ -518,6 +520,8 @@ def _orange_pi_id(self) -> Optional[str]:
         board_value = self.detector.get_device_model()
```

Comment:
Please add the following line under the function def:
`# pylint: disable=too-many-return-statements`